qa/tasks/cephfs/mount.py

```diff
@@ -695,7 +695,7 @@ def run_python(self, pyscript, py_version='python3', sudo=False):
         p.wait()
         return p.stdout.getvalue().strip()
 
-    def run_shell(self, args, timeout=900, **kwargs):
+    def run_shell(self, args, timeout=300, **kwargs):
```

Comment:
Looks reasonable.

Quick question - This doesn't seem to be only effecting fuse/kernel mounts and applies to _every_ command?

Reply:
I am confused by what you meant by "effecting fuse/kernel mounts".

This change affects every command ran using `run_shell()` but not  the commands for mounting through FUSE or kernel because they are mounted using `client_remote.run()`.

Reply:
> I am confused by what you meant by "effecting fuse/kernel mounts".

fuse/kernel related mount commands.

> 
> This change affects every command ran using `run_shell()` but not the commands for mounting through FUSE or kernel because they are mounted using `client_remote.run()`.

Sure.